packages/eslint-plugin/src/rules/naming-convention-utils/enums.ts

```diff
@@ -43,6 +43,9 @@ enum Selectors {
   typeAlias = 1 << 14,
   enum = 1 << 15,
   typeParameter = 1 << 17,
+
+  // other
+  import = 1 << 18,
```

Comment:
I decided it didn't belong to any selector group (I guess `variableLike` comes closest) but feel free to disagree